### ISSUE TITLE
train dev test split with BERT

### ISSUE THREAD

**Issue by ksatvat** [huggingface/transformers#3231]
Does `run_multiple_choice.py` work on train dev test splits?
I need to run BERT on 3 labeled datasets. Train it on my training set, validate it on my validation set (tune hyperparameters and calculate loss), and evaluate it on my test set (report performance). I finally want to do prediction on a forth unlabeled dataset. 
I am wondering which of the codes in your repository includes these 3 modes. Thank you.

**Comment by AdityaSoni19031997** [comment]
You can write or add your own version easily..
